samples/va_intel/va_intel_interop.cpp

```diff
@@ -294,22 +329,50 @@ int main(int argc,char **argv)
     va_status = vaSyncSurface(va::display, surface_id);
     CHECK_VASTATUS(va_status, "vaSyncSurface");
 
-    dumpSurface(va::display, surface_id, argv[1]);
+    dumpSurface(va::display, surface_id, fn1, doInterop);
 
     cv::Size size(CLIP_WIDTH,CLIP_HEIGHT);
     cv::UMat u;
 
-    cv::vaapi::convertFromVASurface(surface_id, size, u);
+    t.start();
+    cv::va_intel::convertFromVASurface(va::display, surface_id, size, u);
     cv::blur(u, u, cv::Size(7, 7), cv::Point(-3, -3));
-    cv::vaapi::convertToVASurface(u, surface_id, size);
+    cv::va_intel::convertToVASurface(va::display, u, surface_id, size);
+    t.stop();
```

Comment:
First OCL run includes OCL kernel creation time. Is it possible to exclude (or separate) first run from time measurement loop?